rascal-lsp/src/main/rascal/util/LanguageServer.rsc

```diff
@@ -268,6 +272,10 @@ data LanguageService
     | references    (set[loc] (Focus _focus) referencesService)
     | implementation(set[loc] (Focus _focus) implementationService)
     | codeAction    (list[CodeAction] (Focus _focus) codeActionService)
+    | prepareRename (loc (Focus _focus) prepareRenameService)
+    | rename        (tuple[list[DocumentEdit], set[Message]] (Focus _focus, str newName) renameService)
```

Comment:
Renaming might want to know about the folders/projects open in the workspace. Currently, `util::LanguageServer` does not expose those. How can we achieve that?

*(also applies to `willRenameFiles`/`didRenameFiles`)*

Reply:
hmm, yes, but that should be a generic thing. This is also something that @rodinaarssen noticed when looking at the `RascalIDEServices` class, that that was missing, so we might have to add that concept of a workspace to rascal.